patches/tModLoader/Terraria/release_extras/LaunchUtils/InstallNetFramework.sh

```diff
@@ -30,6 +30,16 @@ if [ -d "$dotnet_dir" ]; then
 			rm -rf "$old_version"
 		fi
 	done
+	
+	# For some reason, sometimes the dotnet folder is missing some files, if so, delete it so it gets installed again
+	numberFilesInDotnetDir=`find "$install_dir" -type f | wc -l`
+	expectedFilesInDotnetDir=227
```

Comment:
This number would be true strictly for windows x64 binaries. 

Example: Mac x64 binary is 185 files.

Do we want to restrict this to $_uname == _NT for windows only? 
Or do we want to make the fun switch case?

Rest of the code all looks ok, and seems to work fine.